src/status/streamingClient.ts

```diff
@@ -10,7 +10,7 @@ import { resolve as resolveUrl } from 'url';
 import { AsyncOptionalCreatable, Duration, Env, env, set } from '@salesforce/kit/lib';
 import { AnyFunction, AnyJson, ensure, ensureString, JsonMap } from '@salesforce/ts-types/lib';
 // @ts-ignore
```

Comment:
could the ignore be removed now?

Reply:
Not without providing more typing that I'd rather do in v3.  Good eyes!